source/image-handler/setup.py

```diff
@@ -23,11 +23,11 @@
             '': ['*.conf'],
     },
     install_requires=[
-        'botocore==1.3.7',
-        'tornado_botocore==1.0.2',
+        'botocore==1.8',
+        'tornado_botocore==1.3.2',
```

Comment:
`botocore`, `tornado_botocore` as well as `tc_aws` needed to be bumped to get around the `pip` >= `10` issue